packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -635,6 +635,18 @@ class IOSDevice extends Device {
     _logReaders[app] = logReader;
   }
 
+  @visibleForTesting
```

Comment:
Since you use `isDeveloperModeEnabled` is other places, you'll need to remove `@visibleForTesting` to make it public. `@visibleForTesting` basically makes it private but visible to test